pkg/event/logger/logger.go

```diff
@@ -0,0 +1,83 @@
+package logger
+
```

Comment:
Sorry not trying to nit - I mentioned this before - can we rename the file to handler.go for consistency?

Reply:
Sure, sorry for missing some of your comments Tom.